ajax/libs/xcharts/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "xcharts",
+  "filename": "xcharts.min.js",
+  "version": "0.3.0",
+  "description": "A D3-based library for building custom charts and graphs.",
+  "homepage": "http://tenxer.github.io/xcharts/",
```

Comment:
Please try to use https for github.io domain next time, most of them prefer https.